[WIP] Added Dark theme for the graphframe

defnax opened the pull request:
Added Dark theme for the graphframe
Added Combobox for switch

defnax committed:
Added Dark theme for the graphframe

Added Dark theme for the graphframe
Added Combobox for switch

defnax committed:
Fixing some ui heights & wrong values

PhenomRetroShare commented on retroshare-gui/src/gui/common/RSGraphWidget.cpp:
```diff
@@ -345,7 +345,11 @@ void RSGraphWidget::paintEvent(QPaintEvent *)
   
   /* Fill in the background */
-  _painter->fillRect(_rec, QBrush(BACK_COLOR));
+  if (_flags & RSGRAPH_FLAGS_DARK_STYLE){
```
Why not use qproperty so you can change it in qss file?

defnax replied:
i doesnt used so much with qproperity, dont know yet how to format as qproperty, that feature

defnax committed:
Added for other graphs the settings

defnax committed:
Added own graph color settings for the Bandwidth Graph Window

defnax committed:
Added fix by phenom and small ui fixes

Added fix by phenom for get stored the configs for the statistics pages
Removed uneeded line from the toolbar
New icon for Identities stats page
